File: startingPoint/01-EstruturasCondicionais/exercicios_resolvidos/02.py
# 02 - Faça um programa que receba dois números e mostre o menor.
numero1 = int(input("Digite o primeiro número: "))
numero2 = int(input("Digite o segundo número: "))

if numero1 < numero2:
	print("O número {0} é menor que o número {1}.".format(numero1, numero2))
if numero2 < numero1:
	print("O número {0} é menor que o número {1}.".format(numero2, numero1))

File: SRC - PERL/00 - Estruturas Sequenciais/exerProp/02.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);

# Faça um programa que receba dois números, calcule e mostre a divisão do primeiro número pelo segundo. Sabe-se que o segundo número não pode ser zero, portanto, não é necessário se preocupar com validações.

print "Digite o primeiro número >> ";
my $numero1 = <>;
chomp $numero1;

print "Digite o segundo número(não pode ser zero) >> ";
my $numero2 = <>;
chomp $numero2;


my $divisao = $numero1 / $numero2;

printf "%.2f / %.2f = %.2f \n", $numero1, $numero2, $divisao;
